Dockerfile

```diff
@@ -3,6 +3,10 @@
 FROM registry.fedoraproject.org/fedora:35
 WORKDIR /root/containerbuild
 
+# Keep this Dockerfile idempotent for local development rebuild use cases.
+USER root
+RUN rm -rfv /lib/coreos-assembler /usr/bin/coreos-assembler
```

Comment:
Let's use `/usr/lib` for consistency?

Hmm, there's also a bunch of other things in `/usr/bin` we install. Though I guess we can rely on the new build stomping them. Agree `/usr/lib` is the most important.